addons/test_mail/tests/test_mail_scheduled_message.py

```diff
@@ -143,11 +149,28 @@ def test_scheduled_message_posting(self):
             self.assertFalse(self.test_record.message_ids)
             self.assertFalse(self._new_mails)
 
-            with self.mock_datetime_and_now('2022-12-24 14:00:00'):
+            # add a schedule message that will fail to check that it wont block the cron
+            failing_schedueld_message_id = self.schedule_message(
+                test_lead,
+                scheduled_date='2022-12-24 14:00:00',
+                partner_ids=self.test_record.customer_id,
+            ).id
+
+            def _message_post_after_hook(self, message, values):
+                raise Exception("Boum!")
```

Comment:
I click and :boom: !

Reply:
Where did you click for boum?
(maybe it's a ref that I don't have? 😅)

Reply:
It is a reference :D Fabien speech a few years ago

Reply:
Ahhhh yes I get it now 😄